Clicker/Views/Cards/EmptyStateCell.swift

```diff
@@ -18,4 +42,71 @@ class EmptyStateCell: UICollectionViewCell {
         fatalError("init(coder:) has not been implemented")
     }
     
+    func setup() {
+        setupViews()
+        setupConstraints()
+        print("setting up empty state cell with userRole = \(userRole), session=\(session)")
+        // TODO: Add logic for setting up empty state or nonempty state
+        if (userRole == .admin && session.name == session.code) {
+            setupNameView()
+        }
+    }
+    
+    func setupViews() {
+        monkeyView = UIImageView(image: #imageLiteral(resourceName: "monkey_emoji"))
+        monkeyView.contentMode = .scaleAspectFit
+        addSubview(monkeyView)
+        
+        nothingToSeeLabel = UILabel()
+        nothingToSeeLabel.font = ._16SemiboldFont
+        nothingToSeeLabel.textColor = .clickerGrey5
+        nothingToSeeLabel.textAlignment = .center
+        nothingToSeeLabel.text = userRole == .admin ? adminNothingToSeeText : userNothingToSeeText
+        addSubview(nothingToSeeLabel)
+        
+        waitingLabel = UILabel()
+        waitingLabel.font = ._14MediumFont
+        waitingLabel.textColor = .clickerGrey2
+        waitingLabel.textAlignment = .center
+        waitingLabel.lineBreakMode = .byWordWrapping
+        waitingLabel.numberOfLines = 0
+        waitingLabel.text = userRole == .admin ? adminWaitingText : userWaitingText
+        addSubview(waitingLabel)
```

Comment:
lets add these as subviews to `contentView` instead

Reply:
Sure.